BasicsCodelab/build.gradle

```diff
@@ -17,21 +17,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        compose_version = '1.0.0-alpha09'
+        compose_version = '1.0.0-SNAPSHOT'
     }
     ext.kotlin_version = "1.4.21"
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.0-alpha03'
+        classpath 'com.android.tools.build:gradle:7.0.0-alpha04'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     }
 }
 
 allprojects {
     repositories {
+        maven {
+            def snapshot = "7067732"
```

Comment:
I thought the Basics and Layouts codelab were already updated to `alpha10`. Is that not the case?

Reply:
Is alpha10 out already?

Reply:
Sorry, I meant the snapshot of alpha10